lib/state_machine/message.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace hyped::state_machine {
+
+// State class containing all the states
+enum class Message {
+  mForward,
```

Comment:
what is `mForward`? is there a better name for this or is this placeholder messages?

Reply:
looking at the rest of the PR this would be better as something like `kNextNominalState`